php/Terminus/Dispatcher/Subcommand.php

```diff
@@ -177,7 +177,12 @@ private function validate_args( $args, $assoc_args, $extra_args ) {
       exit(1);
     }
 
-    $unknown_positionals = $validator->unknown_positionals( $args );
+    $invalid = $validator->invalid_positionals($args);
+    if($invalid) {
+      \Terminus::error("Invalid positional value: $invalid");
```

Comment:
That's a fully qualified PHP namespace:

http://stackoverflow.com/questions/18095051/in-php-namespaces-a-forward-slash